bin/3llo

```diff
@@ -10,10 +10,29 @@ $container.register(:api_client, Tr3llo::HTTP::Client)
 prompt = TTY::Prompt.new
 $container.register(:interface, Tr3llo::Interface.new(prompt, $stdout))
 
+def print_help(flavour)
+  Tr3llo::Presenter::HelpPresenter
+    .new($container.resolve(:interface), flavour)
+    .print!
+end
+
+unless ARGV.empty?
+  if ARGV[0] =~ /^-h|--help$/
+    print_help(Tr3llo::Presenter::FLAVOUR_HELP_CLI)
+    exit
+  end
+end
+
 configuration = Tr3llo::Configuration.new
-configuration.user_id = ENV.fetch('TRELLO_USER') { raise "Have you set TRELLO_USER?" }
-configuration.api_key = ENV.fetch('TRELLO_KEY') { raise "Have you set TRELLO_KEY?" }
-configuration.api_token = ENV.fetch('TRELLO_TOKEN') { raise "Have you set TRELLO_TOKEN?" }
+begin
+  configuration.user_id = ENV.fetch('TRELLO_USER') { raise "Have you set TRELLO_USER?" }
```

Comment:
So if none of these are set it only errors for the first.
But then again, we don't want an error vomit :smile: